app/assets/stylesheets/module-planification.scss

```diff
@@ -139,17 +139,29 @@ $white: $color_main_negative;
       text-align: right;
     }
 
+    .flex-20 {
```

Comment:
This helpers are quite static. And confusing.
1. It you wanna use helper classes, they should be created with a mixin, to be shared, and to create a lot. In this case, for instance, a loop from 0 through 10
2. The name `flex-10` could be refer to `flex: 10;`, but we have `flex-basis`, instead. Rename it to `flex-basis-10`, `f-b-10`, `flex-b-10`... something like